CHANGES.md

```diff
@@ -1,4 +1,7 @@
 
+## v5.3.4 #######
```

Comment:
I recommend you use a markdown linter, this is not valid heading syntax, it may render but these are unnecessary trailing characters.